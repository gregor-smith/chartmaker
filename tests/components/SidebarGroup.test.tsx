test.todo('renders styled div')
