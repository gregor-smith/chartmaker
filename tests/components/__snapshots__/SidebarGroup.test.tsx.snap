// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders styled div 1`] = `
.emotion-0:not(:last-of-type) {
  border-bottom: 1px solid white;
  padding-bottom: 1rem;
  margin-bottom: 1rem;
}

<div
  id="test-render-container"
>
  <div
    class="emotion-0 test-sidebar-group"
  >
    Test children
  </div>
</div>
`;
